src/app/attendee-survey/page.tsx

```diff
@@ -1,5 +1,180 @@
-export default function Index() {
-    return (
-        <h1>Survey</h1>
-    )
-}
\ No newline at end of file
+
+
+'use client';
+
+import React, { useState } from 'react';
+
+interface IFormData {
+  email: string;
+  name: string;
+  howDidYouHear: string;
+  isFirstEvent: string;
+  favoriteTalk: string;
+  rating: number;
+  feedback: string;
+  joinNextEdition: string;
+}
+
+const SurveyForm: React.FC = () => {
+  const [formData, setFormData] = useState<IFormData>({
+    email: '',
+    name: '',
+    howDidYouHear: '',
+    isFirstEvent: '',
+    favoriteTalk: '',
+    rating: 5,
+    feedback: '',
+    joinNextEdition: '',
+  });
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const { name, value } = e.target;
+    
+    // Rating ko string se number mein convert karne ke liye special handling
+    const finalValue = name === 'rating' ? parseInt(value, 10) : value;
+
+    setFormData((prevData) => ({
+      ...prevData,
+      [name]: finalValue,
+    }));
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log('Survey Form Data:', formData);
+    alert('Thank you for your feedback!');
+    // Yahan par aap data ko backend API par bhej sakte hain
+  };
+
+  return (
+    // Section ka background dark red-black gradient jaisa banaya
+    <section className="py-16 bg-gradient-to-b from-black via-red-950 to-black text-white"> 
```

Comment:
Keep background completely black, remove red gradient. We will add later as per requirements